.gitignore

```
@@ -1,6 +1,7 @@
 __pycache__
 venv
-.doit.db
+.doit.db.*
```

```suggestion
.doit.db.*
.doit.db
```
The old entry has to stay else it will not ignore the original file. I don't have .doit.db.db